types/mempool/nonce.go

```diff
@@ -0,0 +1,125 @@
+package mempool
+
+import (
+	"fmt"
+
+	huandu "github.com/huandu/skiplist"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/signing"
+)
+
+var (
+	_ Mempool  = (*nonceMempool)(nil)
+	_ Iterator = (*nonceMempoolIterator)(nil)
+)
+
+// nonceMempool is a mempool that keeps transactions sorted by nonce. Transactions
+// with the lowest nonce globally are prioritized. Transactions with the same
+// nonce are prioritized by sender address. Fee/gas based prioritization is not
+// supported.
```

Comment:
Does that mean that newer accounts get higher priority? So if I want for my trades to be prioritized, would I be able to keep creating new addresses/accounts?

Reply:
Whoops. Great catch! We refactored this in the core repo to be random sender! Pushing changes.